maintainers/maintainer-list.nix

```
@@ -8806,6 +8806,11 @@
     githubId = 33375;
     name = "Peter Sanford";
   };
+  pshirshov = {
+    email = "pshirshov@eml.cc";
+    github = "pshirshov";
```

```suggestion
    github = "pshirshov";
    githubId = 295225;
```
https://api.github.com/users/pshirshov